pkg/start/start.go

```diff
@@ -195,6 +192,17 @@ func (o *Options) run(ctx context.Context, controllerCtx *Context, lock *resourc
 	controllerCtx.OpenshiftConfigManagedInformerFactory.Start(informersDone)
 	controllerCtx.InformerFactory.Start(informersDone)
 
+	klog.Info("!!!! Waiting for cache to sync...")
+	if !cache.WaitForCacheSync(informersDone) {
+		klog.Errorf("Caches never synchronized: %v", postMainContext.Err())
+		return
```

Comment:
Hmm.  `klog.Fatalf`, or update `run` and above to allow returning an error we can bubble up the stack?

Reply:
Done in  bba8ed4eae.